lib/rucio/db/sqla/migrate_repo/versions/d1189a09c6e0_oauth2_0_and_jwt_feature_support_adding_.py

```diff
@@ -0,0 +1,159 @@
+
+from alembic import context
+from alembic.op import (add_column, alter_column, drop_column, create_table,
+                        create_primary_key, create_index,
+                        create_check_constraint, drop_table, drop_constraint, execute)
+from rucio.db.sqla.types import InternalAccountString
+
+# Alembic revision identifiers
+revision = 'd1189a09c6e0'
+down_revision = '810a41685bc1'
+
+
+def upgrade():
+    '''
+    Upgrade the database to this revision
+    '''
+    schema = context.get_context().version_table_schema + '.' if context.get_context().version_table_schema else ''
+    if context.get_context().dialect.name in ['oracle', 'postgresql']:
+        drop_constraint('IDENTITIES_TYPE_CHK', 'identities', type_='check')
+        create_check_constraint(constraint_name='IDENTITIES_TYPE_CHK',
+                                table_name='identities',
+                                condition="identity_type in ('X509', 'GSS', 'USERPASS', 'SSH', 'SAML', 'OIDC')")
+        drop_constraint('ACCOUNT_MAP_ID_TYPE_CHK', 'account_map', type_='check')
+        create_check_constraint(constraint_name='ACCOUNT_MAP_ID_TYPE_CHK',
+                                table_name='account_map',
+                                condition="identity_type in ('X509', 'GSS', 'USERPASS', 'SSH', 'SAML', 'OIDC')")
+    elif context.get_context().dialect.name == 'mysql' and context.get_context().dialect.server_version_info[0] == 5:
+        create_check_constraint(constraint_name='IDENTITIES_TYPE_CHK',
+                                table_name='identities',
+                                condition="identity_type in ('X509', 'GSS', 'USERPASS', 'SSH', 'SAML', 'OIDC')")
+        create_check_constraint(constraint_name='ACCOUNT_MAP_ID_TYPE_CHK',
+                                table_name='account_map',
+                                condition="identity_type in ('X509', 'GSS', 'USERPASS', 'SSH', 'SAML', 'OIDC')")
+
+    elif context.get_context().dialect.name == 'mysql' and context.get_context().dialect.server_version_info[0] == 8:
+        execute('ALTER TABLE ' + schema + 'identities DROP CHECK IDENTITIES_TYPE_CHK')  # pylint: disable=no-member
+        create_check_constraint(constraint_name='IDENTITIES_TYPE_CHK',
+                                table_name='identities',
+                                condition="identity_type in ('X509', 'GSS', 'USERPASS', 'SSH', 'SAML', 'OIDC')")
+        execute('ALTER TABLE ' + schema + 'account_map DROP CHECK ACCOUNT_MAP_ID_TYPE_CHK')  # pylint: disable=no-member
+        create_check_constraint(constraint_name='ACCOUNT_MAP_ID_TYPE_CHK',
+                                table_name='account_map',
+                                condition="identity_type in ('X509', 'GSS', 'USERPASS', 'SSH', 'SAML', 'OIDC')")
+
+    if context.get_context().dialect.name in ['oracle', 'mysql', 'postgresql']:
+        add_column('tokens', sa.Column('scope', sa.String(2048), nullable=True, default=None), schema=schema[:-1])
```

Comment:
I understand that for 'scope' it could make sense as you use it in another sense in Rucio jargon, on the other hand, we are in token table, so the sense might not be so confusing after all. I imagine that in the future the OIDC will be the default authentication mechanism, do you really want to have all the OIDC columns marked with oidc_ prefix ?

Reply:
OK, Done, the DB column now has name 'oidc_scope'.